src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/Regex.Match.Tests.cs

```diff
@@ -1228,7 +1228,7 @@ public static IEnumerable<object[]> Match_Timeout_Throws_MemberData()
                         // Loop around a single-char loop
                         yield return new object[] { engine, @$"(a+{lazyInner})+{lazyOuter}$", $"{a50}b" };
                         yield return new object[] { engine, @$"([^a]+{lazyInner})+{lazyOuter}$", $"{b50}a" };
-                        yield return new object[] { engine, @$"(\w+{lazyInner})+{lazyOuter}$", $"{a50}!" };
+                        yield return new object[] { engine, @$"(\w+{lazyInner})+{lazyOuter}$", $"{a100}!" };
```

Comment:
I'm curious why this changed?

Reply:
This data is used with the timeout tests. It happened in my machine to process this one faster than 1 ms. So, the test was failing as it didn't throw time out exception.